src/pages/library/DataBrowser.js

```diff
@@ -221,11 +224,33 @@ const Browser = () => {
   const [selectedData, setSelectedData] = useState([])
   const [requestDatasetAccessList, setRequestDatasetAccessList] = useState()
   const { dataCatalog, loading } = useDataCatalog()
+  const [acknowledged, setAcknowledged] = useState(getDynamic(localStorage, acknowledgmentStorageKey))
```

Comment:
For this we may want to use a `staticStorageSlot` as `dynamicStorage` is meant to be used for more transient settings (e.g. navigating between routes and preserving some data). Dynamic storage will be the first to get deleted. Also, by using a `staticStorageSlot` we can utilize the `useStore` hook.

Also, when assigning a key to this value, we should add the user id so we can fetch the correct key if the user logs out and in again as another user, or another user signs in, etc... You can retrieve this with the `getUser` function from `src/libs/auth` via `getUser().id`

Reply:
i think i've updated this to fix this now, please let me know if this works!